tests/sysauth/sssd.pm

```diff
@@ -17,19 +17,33 @@ use warnings;
 use testapi;
 use utils;
 use version;
-
+use version_utils qw(is_sle is_opensuse);
 
 sub run {
     # Assume consoletest_setup is completed
     select_console 'root-console';
+    if (is_sle) {
```

Comment:
I think you can use `registration::add_suseconnect_product`  instead

Reply:
Sure, that what I want to do, but the problem is that it will fail for PackageHub:
see example http://f40.suse.de/tests/3353#step/sssd/13
So we need to do like:
if (script_run('SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300) != 0) {
￼                 record_soft_failure 'bsc#1124318 - Fail to get PackageHub Pool Metadata - running the command again as a workaround';

Reply:
Could you create a ticket as a follow up to this?...
```
   add_suseconnect_product should allow to retry n times.
```

Reply:
yes, created now: https://progress.opensuse.org/issues/50336